content/en/continuous_testing/environments/multiple_env.md

```diff
@@ -137,7 +137,8 @@ The <code>resourceUrlSubstitutionRegexes</code> is also applied to the first req
 </div>
 
 <div class="alert alert-info">
-Apart from the pipe <code>|</code> syntax presented above, <code>resourceUrlSubstitutionRegexes</code> also supports the sed syntax with modifiers: <code>s|&lt;regex&gt;|&lt;rewriting rule&gt;|&lt;modifiers&gt;</code>.</br></br> The sed syntax is often used with a slash <code>/</code> separator, for example: <code>s/&lt;regex&gt;/&lt;rewriting rule&gt;/&lt;modifier&gt;</code>. However, it can use any character as a delimiter. When working on a URL containing an abundant number of slashes, Datadog recommends using another character rather than escaping all slashes of the URL.
+Apart from the pipe <code>|</code> syntax presented above, <code>resourceUrlSubstitutionRegexes</code> also supports the sed syntax: <code>s/&lt;regex&gt;/&lt;rewriting rule&gt;/&lt;modifiers&gt;</code>.</br></br>
+But given this syntax uses a slash <code>/</code> separator, it may require escaping slashes from the URL which is error-prone. Unless you need regex modifiers, Datadog recommends using the pipe <code>|</code> syntax for better readability.
```

Comment:
```suggestion
Because this syntax uses a slash <code>/</code> separator, it may require escaping slashes from the URL, which is error-prone. Unless you need regex modifiers, Datadog recommends using the pipe <code>|</code> syntax for better readability.
```